portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,19 +34,28 @@
     
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-     //retrieves comment data from datastore
+     // Get the user input on how many comments to display
+     
+    Integer maxComments = getMaxComments(request);
+     //retrieves comment data from datastore 
+     //displays most recent comments first
```

Comment:
Obviously this is a bonus feature, but it might be nice if the user can select ascending or descending in the UI.